src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -493,7 +473,7 @@ export default function Swap({ history, location, className }: RouteComponentPro
                 showMaxButton={false}
                 hideBalance={false}
                 fiatValue={fiatValueOutput ?? undefined}
-                priceImpact={onWrap ? undefined : priceImpact}
+                priceImpact={!showWrap ? undefined : priceImpact}
```

Comment:
No, if `showWrap`, we don't want to show the price impact. There should not be a `!` here

Reply:
Fixed in the last PR:
https://github.com/cowprotocol/cowswap/blob/refactor/new-swap-form-p4/src/custom/pages/Swap/SwapMod.tsx#L334